### DISCUSSION TITLE
Plans for Europython 2021?

### DISCUSSION THREAD

**Discussion by The-Compiler** [pytest-dev/pytest#8637] [General]
Hey! Does anyone here have any plans for Europython 2021 related to pytest?

I'm thinking about submitting:

- A 3h training for pytest, somewhat similar on what I did for [PyConDE](https://www.youtube.com/watch?v=CMuSn9cofbI) - but with much nicer slides, and probably with most exercises being converted to live demos, as I don't think this would work out remotely otherwise (with potentially a quite big audience, the PyConDE training was overflowing)
- If nobody else wants to do something, a talk (probably 45min) with a rough introduction about pytest - i.e. the basics of assertions, marks, fixtures, a quick overview over plugins, etc.

Is anyone else planning on doing something? If so, it'd be good to coordinate to avoid duplicating (and I'd be more than happy to not do both training and talk, as I'm also planning to submit a poster for [qutebrowser](https://www.qutebrowser.org/) :sweat_smile:).

Anyone feels like organizing a sprint session on the weekend maybe?

**Comment by nicoddemus** [comment]
I won't be participating, but wish of course best of luck! I'm sure it will be great! 😁
